eventmesh-starter/build.gradle

```diff
@@ -18,5 +18,6 @@
 dependencies {
     implementation project(":eventmesh-runtime")
     testImplementation project(":eventmesh-runtime")
+    implementation project(":eventmesh-connector-plugin:eventmesh-connector-rocketmq")
```

Comment:
@Hu-Tu-Tu It does not need to add plugin to the dependencies, please refer to the plugin load mechanisam.